### PR TITLE
AOM-26: UI improvements and fix location drop down bug

### PR DESCRIPTION
**Pull request by jomadoye**
# JIRA TICKET NAME:
AOM-22: [UI improvements and fix location drop down bug](https://issues.openmrs.org/browse/AOM-26)

## SUMMARY:
This PR ensures that the current location is displayed to the user on the dropdown, as well as other slight UI fixes

### CODE REVIEW

File: app/css/openmrs-addonmanager.css [hunk 1]
```diff
@@ -126,10 +126,10 @@ a:hover {
 }
 
 .dropdown-menu-large>li>a:hover {
-    background-color: transparent;
-    margin-right: 0px;
-    border-bottom: 0px;
-    color: purple;
+    /* background-color: transparent;
```
Comment (paulupendo) [inline app/css/openmrs-addonmanager.css]: Could we not comment out this code and not have it in the codebase if it is not in use.

File: app/js/components/common/header.js [hunk 2]
```diff
@@ -104,27 +122,41 @@ export default class Header extends Component {
...
 
         <ul className="navbar-right nav-header">
-          <Link to="" activeClassName="active">
+          <Link to="" activeClassName="active" className="">
             <li className="dropdown">
-              <a className="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
-                <span className="glyphicon glyphicon-user"/> {' ' + this.state.currentUser}
-                <span className="caret"/>
+              <a 
+                id="jed"
```
Comment (paulupendo) [inline app/js/components/common/header.js:128]: Could we have a more definitive id name rather than 'jed'

**Review by paulupendo — COMMENTED** [review]
Good job @andela-jomadoye

**Comment by jomadoye** [comment]
Thanks for the review @paulupendo the PR is still in progress.

**Comment by wanjikum** [comment]
@andela-jomadoye, did you see the merge conflicts?

**Pull request closed without merging**